Functions.Templates/Templates/TimerTrigger-CSharpExcel/metadata.json

```diff
@@ -0,0 +1,9 @@
+﻿{
+  "defaultFunctionName": "TimerTriggerCSharpExcel",
+  "description": "$Read two Excel tables, send emails based on the data, and then update an Excel table.",
+  "name": "O365 Demo",
```

Comment:
replace demo with something descriptive. This is the template name displayed in the portal